.travis.yml

```diff
@@ -26,3 +47,5 @@ script:
   - make
   - if [ $USE_CMAKE == "YES" ] ; then CTEST_OUTPUT_ON_FAILURE=1 make test ; fi
   - if [ $USE_CMAKE == "NO" ] ; then make check ; fi
+  - cppcheck --language=c --std=c89 --verbose --quiet --error-exitcode=1 --enable=warning,performance,portability src/
```

Comment:
Is there a way to organize this file so that the cppcheck is only run once instead of against {linux,osx} x {gcc,clang} x {autotools, cmake} instead of for all possible combinations. From what I understand how the code is compiled would not matter, as this analyzes the source code instead of binaries.